frontend/src/queries/schema.ts

```diff
@@ -787,6 +787,7 @@ export type PathsFilter = {
     maxEdgeWeight?: PathsFilterLegacy['max_edge_weight']
     funnelPaths?: PathsFilterLegacy['funnel_paths']
     funnelFilter?: PathsFilterLegacy['funnel_filter']
```

Comment:
Isn't it required for legacy? Or maybe I lost track what is.

Reply:
Yep, it's required for legacy. The `schema.ts` file generates the non-legacy query nodes (whereas the `FunnelsFilterType` filters come from `types.ts`. Since the new query node already contains all info from the `funnelsFilter` in the `funnelActorsQuery`, we can use that in the `filterToQuery` conversion function.

Hope I'm not confusing you more with this word salad 😅